spec/support/models/name.cr

```diff
@@ -0,0 +1,5 @@
+class Name < BaseModel
+  view :all_the_names, materialized: true do
+    column name : String
+  end
```

Comment:
These technically fail on the SchemaEnforcer... Should materialized views call `skip_schema_enforcer` by default, or would it be better to figure out why they fail?